### ISSUE TITLE
CLP Genesis 

### ISSUE THREAD

**Issue by intl-man** [Sifchain/sifnode#175]
When generating a new genesis file, the following error is returned:

```
ERROR: failed to validate genesis state: failed to unmarshal clp genesis state: UnmarshalJSON cannot decode empty bytes
```

as it appears that the initial CLP Genesis state is emtpy.

**Comment by utx0** [comment]
@Jedi2002 I think we have a need in clp to make sure the genesis export state is exported correctly and completely. Otherwise we could have big upgrade issues. cc @jkbrooks @AustinoBombino @musnit. We had to commect some of the code out as this was blocking us from deployments.

**Comment by musnit** [comment]
yeh i think you're right

**Comment by intl-man** [comment]
@utx0 @musnit On further investigation, this was my fault - I was marshalling genesis with an old (pre-CLP) struct. Since been fixed and merged (#177).
